research/fed-sm/plot-requirements.txt

```diff
@@ -0,0 +1,3 @@
+tensorflow
```

Comment:
Could be merged into one requirements.txt

Reply:
merged